src/controllers/UserController.ts

```diff
@@ -72,6 +76,23 @@ export class UserController {
 		}
 	}
 
+	public async reportUser(req: Request & { params: { id: string } }, res: AuthenticatedResponse, next: NextFunction): Promise<void> {
+		try {
+			if (!req.params.id) throw new APIError(HttpCode.NotFound, GetUserError.UserNotFound);
+			if (req.params.id === '@me') throw new APIError(HttpCode.NotFound, ReportUserError.ReportInvalid);
```

Comment:
So this check here isn't necessary, since it will fail below if `@me` is passed.